docker-compose.yaml

```diff
@@ -13,6 +13,9 @@ services:
       - "--dev"
     networks:
       - internal
+    healthcheck:
+      test: ["CMD", "echo", "ok"]
```

Comment:
can we just add a comment saying that this is a hack waiting for madara to implement something better?

Reply:
I think that Oak has added a HEALTHCHECK in the madara image, tbc but we could then remove this

Reply:
Indeed there is this pr https://github.com/keep-starknet-strange/madara/pull/915, I've asked Madara team to push the new image so we can just use that 👍